_posts/2022-02-07-sparse-models.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Speeding up Inference with the Lottery Ticket Hypothesis
```

Comment:
General comment: What are we adding more than what's been already said about lottery ticket hyp?

Reply:
The later part of the blog are results from my recent paper. I thought it fits into the article, hence included it.

I'll remove it if sounds irrelevant.

Reply:
Let me know how much technicality should be there/purpose of blogs are so that I can better tailor towards it.